### PR TITLE
Form controller

### PR DESCRIPTION
**Pull request by ctlong**
Added a controller and views for forms.

### CODE REVIEW

File: app/controllers/forms_controller.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
+class FormsController < ApplicationController
+  def index
+    @forms = User.find(params[:user_id]).forms.includes(:contents)
```
Comment (h8rry) [inline app/controllers/forms_controller.rb]: You can use `current_user` here

File: app/controllers/forms_controller.rb [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  end
+
+  def create
+    @form = User.find(params[:user_id]).forms.create(form_params)
```
Comment (h8rry) [inline app/controllers/forms_controller.rb]: You can use `current_user` here

**Comment by h8rry** [comment]
@ctlong Please review

**Comment by ctlong** [comment]
@ggharry Reviewed and modified.

File: app/controllers/forms_controller.rb [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+      render json: {updated: true}, status: :success
+    end
+  end
+
+  def delete
+    @form = Form.find(params[:id])
+    if @form.nil?
+      render json: {message: "Cannot find form"}, status: :not_found
+    else
+      @form.destroy
```
Comment (h8rry) [inline app/controllers/forms_controller.rb]: same as https://github.com/WDI-HK-8/Formidable-web/pull/7#discussion_r39002702

**Pull request merged**